hibernate-core/src/main/java/org/hibernate/dialect/CockroachDialect.java

```diff
@@ -413,6 +414,11 @@ public void initializeFunctionRegistry(FunctionContributions functionContributio
 		functionFactory.listagg_stringAgg( "string" );
 		functionFactory.inverseDistributionOrderedSetAggregates();
 		functionFactory.hypotheticalOrderedSetAggregates_windowEmulation();
+
+		functionContributions.getFunctionRegistry().register(
+				"trunc", new PostgreSQLTruncRoundFunction( "trunc", true )
```

Comment:
```suggestion
				"trunc", 
				// CockroachDB didn't support the two-argument trunc before version 22.2
				new PostgreSQLTruncRoundFunction( "trunc", getVersion().isSameOrAfter( 22, 2 ) )
```

Reply:
Oops, must have accidentally removed the version check while merging. Thanks!